scripts/docker/dockerfile_gcc-8

```diff
@@ -1,25 +1,38 @@
 FROM axom/compilers:gcc-8
+LABEL maintainer="white238@llnl.gov"
+ARG branch=develop
 
+SHELL ["/bin/bash", "-c"]
 RUN sudo apt-get update -y
-RUN sudo apt-get install curl -fy
-
+RUN sudo apt-get install curl wget ssh pkg-config build-essential libopenblas-dev tar python3 -fy
 RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
```

Comment:
Probably outside the scope of this PR, but any reason the gcc-8 compiler doesn't have gfortran?

Reply:
Also, is there a reason we're using gfortran-7 with our gcc-8 compiler image?

Reply:
not sure if this was copied from somewhere else.  Now that it actually is in a good state we can look into things like this.

Reply:
#403